editor/gui/scene_tree_editor.cpp

```diff
@@ -115,7 +115,7 @@ void SceneTreeEditor::_cell_button_pressed(Object *p_item, int p_column, int p_i
 		}
 		undo_redo->commit_action();
 	} else if (p_id == BUTTON_PIN) {
-		if (n->is_class("AnimationPlayer")) {
+		if (n->is_class("AnimationPlayer") || n->is_class("AnimationTree") || n->is_class("AnimationManager")) {
```

Comment:
```suggestion
		if (n->is_class("AnimationManager")) {
```
`is_class()` already checks for base class.